src/components/post/style.css

```diff
@@ -47,4 +47,16 @@
 
 .post-interactions-container p {
     text-align: right;
+}
+
+.onHover:hover {
+    background-color: #f5faff;
+    border-radius: 5px;
+    cursor: pointer;
+}
+
+button {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+   column-gap: 8px;
```

Comment:
Your CSS changes have made the UI of the app wonky. See screenshots

![cap1](https://user-images.githubusercontent.com/5518649/217226408-9075bd64-3461-4c84-b399-c62cf4e15b44.PNG)
![cap2](https://user-images.githubusercontent.com/5518649/217226412-16b3b890-e21a-4974-bceb-0484e40f16b2.PNG)